redhat-cop/casl-ansible
# Fix #310, byo node gluster var fix

#### What does this PR do?
Fixes issue #310 by not requiring `cns_node_glusterfs_volume` when `glusterfs_devices` is already set.

#### How should this be manually tested?
Using byo sample as a base:
1. Do not set `cns_node_glusterfs_volume` in `host_vars/localhost`
2. Add `[glusterfs]` and `[glusterfs_registry]` with appropriate nodes per OCP 3.10 docs
3. Run inventory against byo-infra.yml playbook
4. Note that casl-ansible fails on pre-install.yml (fails to set `glusterfs_devices` fact)

Solution:
1. Merge this PR
2. Change nothing from above failure procedures
3. Note that casl-ansible continues as expected.

#### Is there a relevant Issue open for this?
#310 

#### Who would you like to review this?
cc: @redhat-cop/casl
